### PR TITLE
Account abstraction

### PR DESCRIPTION
**Pull request by rahul-thirdweb**
Using account abstraction terminology across docs instead of smart wallets

### CODE REVIEW

**Comment by MananTank** [comment]
slug is changed for a few pages - Setup redirects

**Comment by MananTank** [comment]
@joaquim-verges Can you review the content changes?

File: src/app/connect/account-abstraction/faq/page.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Can users have the same smart account contract address on all EVM chains?
+
+Yes, by using our Account factory contracts, it is possible to enable predictable addresses per user across all EVM chains. To do this, ensure that your **account factory** has the same address on every chain. You can accomplish this from the dashboard by enabling "Deterministic address" when deploying the account factory.
+
+### Can I sign transactions with my smart account?
+
+Smart contract wallets cannot sign transactions themselves, only the 'owners' of the smart account can. The way to achieve this is to follow [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271) which requires to ask the smart contract wallet if the recovered address is indeed an owner.
+
+### How do I sign transactions with my smart account?
```
Comment (joaquim-verges) [inline src/app/connect/account-abstraction/faq/page.mdx:23]: can probably remove this, pretty much a repeat of the point above

File: src/app/connect/account-abstraction/faq/page.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { createMetadata } from "@doc";
+
+# FAQs
```
Comment (joaquim-verges) [inline src/app/connect/account-abstraction/faq/page.mdx:3]: this is great, thanks for adding this page

**Pull request merged**